docs/navigation-advanced.md

```diff
@@ -98,7 +98,7 @@ navigation: {
 ### Dynamic viewUrl parameters
 
 You have the following options to add a parameter to **viewUrl**:
-- Place the parameter anywhere in the **viewUrl** value. For example, if the main application URL is `https://[YOUR.WEBSITE]/home/users/JohnSmith`, then the **viewUrl** of the micro frontend in the content area can be `https://example.com/users/details.html#id=JohnSmith`.
+- Place the parameter anywhere in the **viewUrl** value. For example, if the main application URL is `https://[YOUR.WEBSITE]/home/users/:id`, then the **viewUrl** of the micro frontend in the content area can be `https://example.com/users.html#id=JohnSmith`.
```

Comment:
this change is a bit confusing, since it removes the link between main app route and mf url. Should be either JohnSmith as before or ':id' on BOTH URLs. If you choose :id, add additional info that this is then replaced by a real value, e.g. JohnSmith